isamples_metadata/GEOMETransformer.py

```diff
@@ -423,17 +438,92 @@ def curation_responsibility(self) -> typing.AnyStr:
 
     # endregion
 
-    def related_resources(self) -> typing.List[typing.Dict]:
+    def _get_children(self) -> typing.List[typing.Dict]:
+        children = []
         if "children" in self.source_record:
-            related_resources = []
             children = self.source_record["children"]
-            for child in children:
-                child_resource = {}
-                entity = child["entity"]
-                if entity == "Tissue":
-                    child_resource["label"] = "subsample tissue"
-                    child_resource["relationship"] = "subsample"
-                    child_resource["target"] = child["bcid"]
-                    related_resources.append(child_resource)
-            return related_resources
+        return children
+
+    def related_resources(self) -> typing.List[typing.Dict]:
+        related_resources = []
+        for child in self._get_children():
+            child_resource = {}
+            entity = child["entity"]
+            if entity == TISSUE_ENTITY:
+                child_resource["label"] = "subsample tissue"
+                child_resource["relationship"] = "subsample"
+                child_resource["target"] = child["bcid"]
+                related_resources.append(child_resource)
+        return related_resources
+
+    @property
+    def child_transformers(self) -> typing.List[GEOMEChildTransformer]:
+        return self._child_transformers
+
+
+class GEOMEChildTransformer(GEOMETransformer):
+    """GEOME child record subclass transformer -- uses some fields from the parent and some from the child"""
+
+    def __init__(self, source_record: typing.Dict, child_record: typing.Dict):
+        self.source_record = source_record
+        self.child_record = child_record
+
+    def _id_minus_prefix(self) -> typing.AnyStr:
+        return self.child_record["bcid"].removeprefix(self.ARK_PREFIX)
+
+    def sample_label(self) -> typing.AnyStr:
+        return self.child_record["tissueID"]
+
+    def sample_identifier_string(self) -> typing.AnyStr:
+        return self.child_record["bcid"]
+
+    def sample_description(self) -> typing.AnyStr:
+        # TODO
+        return ""
+
+    def has_specimen_categories(self) -> typing.List[typing.AnyStr]:
+        return ["Organism part"]
+
+    def produced_by_label(self) -> typing.AnyStr:
+        return f"tissue subsample from {self._source_record_main_record()['materialSampleID']}"
+
+    def produced_by_description(self) -> typing.AnyStr:
+        description_pieces = []
+        self._transform_key_to_label(
+            "tissueCatalogNumber", self.child_record, description_pieces
+        )
+        return " | ".join(description_pieces)
+
+    def produced_by_feature_of_interest(self) -> typing.AnyStr:
+        return ""
+
+    def produced_by_responsibilities(self) -> typing.List[typing.AnyStr]:
+        # TODO: who did the tissue extract, if available -- where does this live, if anywhere?
```

Comment:
@qgan7125 @YuxuanZhou1 @smrgeoinfo -- any of you know where this might come from?  I didn't see anything in the records.